doc/src/index.rst

```diff
@@ -14,20 +15,73 @@ If you are new to SymPy, start with the :ref:`Tutorial <tutorial>`.
 
 This is the central page for all of SymPy's documentation.
 
+==================================================================
+
+..  rst-class:: column
+
+:ref:`Getting started <getting_started>`
+-------------------------------------------------------
+
+An introduction to SymPy for developers and detailed tutorials.
+
+..  rst-class:: column
+
+:ref:`Tutorials <tutorial>`
+-------------------------------------------------------
+
+A collection of SymPy tutorials.
+
+
+..  rst-class:: column
+
+:ref:`How-to Guides <how_to_guide>`
+-------------------------------------------------------
+
+Covers step-by-step intructions on how to do different key developer tasks.
+
+
+..  rst-class:: column clearfix
+
+:ref:`Explanation <explanation>`
+-------------------------------------------------------
+
+Clarification and discussion of common pitfalls and advanced topics.
+
+
+..  rst-class:: column
+
+:ref:`Reference <reference>`
+-------------------------------------------------------
+
+Discusses and details the internal and public modules
+
+
+..  rst-class:: clearfix row custom-headings
+
+=================================
+
+
+:ref:`Contributing <contributing>`
+-------------------------------------------------------
+
+Covers instructions for SymPy contributors.
```

Comment:
If we have this section, will you be moving content from the wiki here? As that is where it is housed now.

Reply:
As discussed with @asmeurer and @oscarbenjamin , it's my understanding that we are not moving things from the wiki. In my initial audit, I suggested doing so but I was told it's out of scope for the project.

Reply:
Yes, we should make room for stuff eventually being moved from the wiki, so that when it happens we have a place for it to go. But moving it is if not out of scope at least the lowest priority thing in this project, compared to cleaning up what is already in the docs.

Reply:
Maybe at least issues could be opened up to encourage people to build out these new categories, particularly with wiki content. It would make for some nice first contributor issues.